polaris-for-vscode/package.json

```
@@ -1,9 +1,22 @@
 {
   "name": "polaris-for-vscode",
   "displayName": "polaris-for-vscode",
-  "description": "Polaris Design System vscode tooling",
-  "repository": "https://github.com/Shopify/polaris-react",
+  "publisher": "Shopify",
+  "description": "Polaris Design System VSCode extension",
+  "icon": "polaris-for-vscode/docs/polaris-icon.png",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/Shopify/polaris"
```

```suggestion
    "url": "https://github.com/Shopify/polaris.git"
```
Not sure if this update is required, but it is in the [manifest example](https://code.visualstudio.com/api/references/extension-manifest#example).